laborator/content/etapele-compilarii/README.md

```diff
@@ -443,6 +443,9 @@ Editați fișierul `main.c` pentru a apela corespunzător interfața expusă și
 price is 21
 quantity is 42
 ```
+> **NOTE:**
+> Pentru a putea folosi comanda `make` la acest exercițiu va trebui să completați fișierul `Makefile` pentru a realiza linkarea cu interfața
+> `shop.o` (trebuie tratată ca o bibliotecă externă)
```

Comment:
```suggestion
> Pentru a putea folosi comanda `make` la acest exercițiu va trebui să completați fișierul `Makefile` pentru a realiza linkarea cu fișierul obiect
> `shop.o`
```
If you were to treat it as a library, you would have to add `-L. -lname` etc. It's just an object file.